test/basic.js

```diff
@@ -3,41 +3,44 @@ var toBlob = require('../')
 var toBuffer = require('blob-to-buffer')
 var toStream = require('string-to-stream')
 
-test('basic usage', function (t) {
-  t.plan(3)
+test('basic usage with Promise', function (t) {
   var str = '0123456789'
   var stream = toStream(str)
-  toBlob(stream, 'text/plain', function (err, blob) {
-    t.error(err)
-    toBuffer(blob, function (err, buf) {
-      t.error(err)
-      t.equal(buf.toString(), str)
+  toBlob(stream, 'text/plain')
+    .then(function (blob) {
+      toBuffer(blob, function (err, buf) {
+        t.error(err)
+        t.equal(buf.toString(), str)
+        t.end(err)
+      })
     })
-  })
+    .catch(function (err) { t.error(err) })
```

Comment:
If this function is called at all, we want to fail, regardless of the whether `err` is truthy or falsey, so this should use `t.fail()`.